.github/workflows/build_test.yml

```diff
@@ -8,11 +8,34 @@ on:
   pull_request:
     branches: [ main, develop ]
     types: [ opened, synchronize, reopened ]
-        
+ 
 jobs:
   build-test:
     runs-on: ubuntu-latest
     environment: dev
+    env:
+      # Note: The database name may be overridden by the tests
+      CONNECTION_STRING: 'Server=localhost,1433;Database=integrationtests;User Id=sa;Password=P1swrd!$;TrustServerCertificate=True'
```

Comment:
I know this is localhost and running in a container, but should we be picking up the connection string from config anyway as good practice? Potentially linters / code checkers will alert on this.

Reply:
Will move to a secret